modules/apps/frontend-theme/frontend-theme-dialect/src/css/dialect/variables/_navs.scss

```diff
@@ -0,0 +1,387 @@
+// nav-base-link
+
+$nav-base-link: (
+	after: (
+		bottom: 0,
+		content: none,
+		display: block,
+		position: absolute,
+		left: 0,
+		right: 0,
+		width: auto,
+	),
+);
+
+// Primary tab
+
+$nav-underline-link: (
+	active-class: (
+		background-color: var(--tab-primary-item-background-color-active),
+		border-color: var(--tab-primary-item-border-color-active),
+		box-shadow: var(--tab-primary-item-box-shadow-active),
+		color: var(--tab-primary-item-color-active),
+		font-weight: var(--tab-primary-item-font-weight-active),
+		after: (
+			background-color: var(--tab-primary-item-active-bar-color-active),
+			height: var(--tab-primary-item-active-bar-thickness),
+			content: '',
+		),
+		inline-item: (
+			color: var(--tab-primary-item-icon-color-active),
+		),
+	),
+	after: (
+		border-radius: var(--tab-primary-item-active-bar-border-radius),
+		height: var(--tab-primary-item-active-bar-thickness),
+		width: var(--tab-primary-horizontal-item-active-bar-width),
+	),
+	background-color: var(--tab-primary-item-background-color),
+	border-color: var(--tab-primary-item-border-color),
+	border-radius: var(--tab-primary-item-border-radius),
+	border-style: solid,
+	border-bottom-width: var(--tab-primary-horizontal-item-border-bottom-width),
+	border-left-width: var(--tab-primary-horizontal-item-border-left-width),
+	border-right-width: var(--tab-primary-horizontal-item-border-right-width),
+	border-top-width: var(--tab-primary-horizontal-item-border-top-width),
+	color: var(--tab-primary-item-color),
+	disabled: (
+		border: 0,
+		after: (
+			content: none,
```

Comment:
Is this necessary? If we don't want the underline to be displayed is there a way to do it by removing css rather than by adding it. For instance I think you could remove background color from here using https://clayui.com/sass-api/index.html#variable-clay-unset

Reply:
Whenever possible we want to reduce our css size to make it faster. :)

Reply:
This was discussed in the last meeting